.azure-pipelines/scripts/junit_utils.sh

```diff
@@ -136,12 +136,12 @@ function JunitTestFinished()
         fi
 	# Replace STACK_TRACE place holder with stack trace file
         AddLogFileToJunit "$stack_trace_file_path" "$junit_file_path" "STACK_TRACE"
-    fi
 
-    # Add stdout file to junit if exist
-    if [[ -f "$stdout_file_path" ]]; then
-        echo "      <system-out>STD_OUT_MESSAGE</system-out>" >> "$junit_file_path"
-        AddLogFileToJunit "$stdout_file_path" "$junit_file_path" "STD_OUT_MESSAGE"
+        # Add stdout file to junit if exist
```

Comment:
This is not related to nightly build, right?

Reply:
Yes this not directly related but there is an issue with downloaded html page encoding with basic/networking test that blocks junit xml file and all core test junit file publish. To be able to see test results I need to fix this. This can be a separate PR but if it is OK I want to fix it as part of this PR.